.github/workflows/shellcheck-ignored.yaml

```diff
@@ -0,0 +1,16 @@
+# Test our shell scripts for bugs
```

Comment:
I suggest adding a comment to the new `-ignored` workflows, why we need them.

Reply:
Added a comment to the beginning of each file.

Reply:
Too bad github still doesn't support early exit: https://github.com/actions/runner/issues/662
We could've done without extra files if we could use early exit and changed paths filter (https://github.com/marketplace/actions/paths-changes-filter)